app/main.py

```diff
@@ -1 +1,65 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        front_skills = [
+            "JavaScript",
+            "HTML",
+            "CSS",
+        ]
+        for skill in front_skills:
+            self.learn_skill(skill)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        back_skills = [
+            "Python",
+            "SQL",
+            "Django",
+        ]
+        for skill in back_skills:
+            self.learn_skill(skill)
+
+    def create_powerful_api(self) -> str:
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        self.name = name
+        android_skills = [
+            "Java",
+            "Android studio",
+        ]
+        for skill in android_skills:
+            self.learn_skill(skill)
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
```

Comment:
what are skills of full stack dev?

Reply:
learn multiple inheritance )

Reply:
learn inheritance ) 
![image](https://user-images.githubusercontent.com/107141441/218155382-227eb90c-5094-43d9-9189-e5803436538e.png)

Reply:
@NEkropo1, please, be a bit more kind, when answering questions :)

Reply:
Didn't try to insult, it was just a simple suggestion in my opinion and a screenshot of result... Mb need a little bit more understanding of politeness  (